server/pbench/bin/pbench-rsync-satellite-process-work-queue

```
@@ -61,7 +60,10 @@ def process(todo, options):
         if options.verbose:
             print("os.symlink({}, {})".format(os.path.join(x.archive, x.name), x.link_name))
         if not options.dryrun:
-            os.symlink(os.path.join(x.archive, x.name), x.link_name)
+            try:
+                os.symlink(os.path.join(x.archive, x.name), x.link_name)
+            except OSError:
```

Should we report the error encountered?